helm/ffc-demo-calculation-service/templates/deployment.yaml

```diff
@@ -34,40 +34,40 @@ spec:
             {{- toYaml . | nindent 12 }}
         {{- end }}
         {{- end }}
-        imagePullPolicy: {{ .Values.container.imagePullPolicy }}
+        imagePullPolicy: {{ quote .Values.container.imagePullPolicy }}
         securityContext:
           runAsUser: {{ .Values.container.runAsUser }}
           runAsNonRoot: {{ .Values.container.runAsNonRoot }}
           readOnlyRootFilesystem: {{ .Values.container.readOnlyRootFilesystem }}
           allowPrivilegeEscalation: {{ .Values.container.allowPrivilegeEscalation }}
         env:                
         - name: NODE_ENV
-          value: {{ .Values.environment }}
+          value: {{ quote .Values.environment }}
         - name: MESSAGE_QUEUE_HOST
-          value: {{ .Values.container.messageQueueHost }}
+          value: {{ quote .Values.container.messageQueueHost }}
         - name: MESSAGE_QUEUE_PORT
-          value: {{ .Values.container.messageQueuePort | quote }}
+          value: {{ .Values.container.messageQueuePort }}
```

Comment:
Judging from the error in Jenkins, it seems that the port does need to be quoted.